run.sh

```diff
@@ -58,7 +58,7 @@ case "$deployment_action" in
         deploy_osh
         ;;
     "add_compute")
-        add_compute
+        add_compute add_computenode
```

Comment:
Is there a need to pass add_computenode ? I mean are there different actions possible for add_compute function ?